src/main/scala/viper/silver/plugin/standard/inline/InlinePredicatePlugin.scala

```diff
@@ -0,0 +1,42 @@
+package viper.silver.plugin.standard.inline
+
+import viper.silver.ast.{Exp, LocalVar, Method, Node, PredicateAccess, Program}
+import viper.silver.plugin.{ParserPluginTemplate, SilverPlugin}
+
+class InlinePredicatePlugin extends SilverPlugin with ParserPluginTemplate {
+
+  private[this] val InlinePredicateKeyword = "inline"
+
+  override def beforeVerify(input: Program): Program = {
```

Comment:
This is the hook we run before actual verification occurs. So we'll need to do any AST transformations here.